modules/preview-react/status-indicator/lib/StatusIndicator.tsx

```diff
@@ -13,13 +16,95 @@ export interface StatusIndicatorProps extends ExtractProps<typeof Flex, never> {
   children: React.ReactNode;
 }
 
-export const useStatusIndicator = createElemPropsHook(useStatusIndicatorModel)(({state}) => {
-  const colors = statusIndicatorColors[state.variant][state.emphasis];
-  return {
-    opacity: state.variant === 'transparent' ? '0.85' : undefined,
-    color: colors.text,
-    background: colors.background,
-  };
```

Comment:
This is a breaking change that we should add to the upgrade guide

Reply:
Oh, yeah. It is. I'm surprised it's exported.

Reply:
I added the back, can you please take a look?